.lando.yml

```diff
@@ -5,44 +5,51 @@ recipe: wordpress
 config:
   php: '7.2'
   via: nginx
-  webroot: trunk
+  webroot: core/build
   database: mariadb
-  xdebug: false
+  xdebug: true
+
+  conf:
+    server: config/wordpress.conf
 
 services:
   appserver:
     composer:
+      # Note installing phpunit should be temporary because core should really install it via composer on its own.
       phpunit/phpunit: "^6"
+    install_dependencies_as_me:
+      - if [ ! -e $LANDO_MOUNT/core ]; then git clone --depth 1 https://github.com/WordPress/wordpress-develop.git $LANDO_MOUNT/core; fi
+      - if [ ! -e $LANDO_MOUNT/core/wp-config.php ]; then cp $LANDO_MOUNT/config/wp-config.php $LANDO_MOUNT/core/wp-config.php; fi
+      - if [ ! -e $LANDO_MOUNT/core/wp-tests-config.php ]; then cp $LANDO_MOUNT/config/wp-tests-config.php $LANDO_MOUNT/core/wp-tests-config.php; fi
+      - mkdir -p $LANDO_MOUNT/content/plugins
+      - mkdir -p $LANDO_MOUNT/content/themes
+      - if [ ! -e $LANDO_MOUNT/content/themes/twentynineteen ]; then cp -r $LANDO_MOUNT/core/src/wp-content/themes/twentynineteen $LANDO_MOUNT/content/themes/twentynineteen; fi
+      - cd $LANDO_MOUNT/core && composer install
     install_dependencies_as_root:
-      - apt-get update -y
-      - apt-get install gnupg -y
-      - curl -sL https://deb.nodesource.com/setup_8.x | bash -
-      - apt-get install subversion -y
-      - apt-get install nodejs -y
```

Comment:
This should be restored instead of relying on a separate `node` service because PHP and JS need to be available in the same env.